pkg/e2e/harness_runner/harness_runner.go

```diff
@@ -87,6 +88,12 @@ var _ = ginkgo.Describe("Test harness", ginkgo.Ordered, ginkgo.ContinueOnFailure
 			Expect(err).NotTo(HaveOccurred(), "Could not read results")
 			ginkgo.By("Writing results")
 			h.WriteResults(results)
+			if config.Tests.LogBucket != "" {
+				err = h.UploadResultsToS3(results, harnessImage+time.DateTime)
```

Comment:
`time.DateTime` is a constant, I don't think that is what you are wanting to use https://pkg.go.dev/time#pkg-constants

(note the name of your S3 buckets, the date is 2006)